Frontend/ui-customer/src/Flow.purs

```diff
@@ -2938,8 +2938,9 @@ metroTicketBookingFlow = do
   metroStationsList <- lift $ lift $ getMetroStationsObjFromLocal ""
   let diffSec = runFn2 differenceBetweenTwoUTCInMinutes (getCurrentUTC "") metroStationsList.lastUpdatedAt
       metroStationValidTill  = config.metroTicketingConfig.metroStationTtl
+  let currentCity = getValueToLocalStore CUSTOMER_LOCATION
```

Comment:
@VD321 Logic to cache needs to be refactored for different city/station usecase

Reply:
Done